components/SensorComponent.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect } from "react";
+import { View, Image, StyleSheet } from 'react-native';
+import { Accelerometer } from "expo-sensors";
+import styled from "styled-components/native";
+
+const styles = StyleSheet.create({
+
+  Image: {
+    width: 350,
+    height: 350,
+    borderRadius: 10,
+  },
+});
+
+
+export const SensorComponent = () => {
+  const [image, setImage] = useState({})
+
+  const generateImage = () => {
```

Comment:
The pictures of the dogs are so cute!! This entire component is full with code but you manage to modify and structure it well so it gets easy to follow. I am happy that you found your way to add a new API address and apply that to the fetch. Now you have any problems finding and using API:s in your future projects. Small action but great impact!